fms/distributed/tensorparallel.py

```diff
@@ -144,30 +142,30 @@ class _ReduceFromModelParallelRegion(torch.autograd.Function):
     """All-reduce the input from the model parallel region."""
 
     @staticmethod
-    def symbolic(graph, input_):
-        return _all_reduce(input_)
+    def symbolic(graph, input_, world_size):
+        return _all_reduce(input_, world_size)
 
     @staticmethod
-    def forward(ctx, input_):
-        return _all_reduce(input_)
+    def forward(ctx, input_, world_size):
+        return _all_reduce(input_, world_size)
 
     @staticmethod
     def backward(ctx, grad_output):
-        return grad_output
+        return grad_output, None
```

Comment:
is anyone using this that a different output would break?

Reply:
I couldn't find it. @ani300 ?

Reply:
We need this extra output for pytorch compile reasons

Reply:
This function is never called directly by users

Reply:
Thanks!